docs/assets/img/BadgeSilver.svg

```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" class="svg-icon iconBadgeSilver native" width="16" height="16" viewbox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path fill="#8C9298" fill-rule="evenodd" d="M8 5c1.657 0 3 1.343 3 3s-1.343 3-3 3-3-1.343-3-3 1.343-3 3-3"/></svg>
```

Comment:
These won't have the `svg-icon icon[X] native` in production, so we should kill those classes and make sure they're still showing up how we want them to via `.s-badge--image`

Reply:
I was unsure how to do this because I need `native` so `svg-icon` CSS doesn't re-color them. These would be delivered by `Design/Icons` I would imagine, correct?

Reply:
Just remove all the classes and they should display in their native colors. These won't be delivered by `Design/Icons`. They'll be static assets in our CDN within the sites folder.